libs/positions/src/lib/positions-manager.tsx

```diff
@@ -19,6 +18,8 @@ interface PositionsManagerProps {
   showClosed?: boolean;
 }
 
+const MAXINT64 = '9223372036854775807';
```

Comment:
We use the same constant for deposit approvals. Perhaps we can add this to utils and share the value.

Can we also add some comments saying why we use this value

Reply:
Only we need there max 64 bit int for golang, and in deposit we have max 256 bit uint from ethereum lib, and I can't see there is int64 available. So I'll just move it to some constants file.

Reply:
is it worth getting a test in for deposit approval as well if we are building this into utils?

Reply:
> is it worth getting a test in for deposit approval as well if we are building this into utils?

I think deposit will remain like it is now, with uint256 from ethereum and close position tx with int64 stored in the utilis lib